<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editions</class>
 <widget class="QWidget" name="editions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1077</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(230, 230, 250);</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>20</y>
     <width>601</width>
     <height>461</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="name_LineEdit">
      <property name="font">
       <font>
        <family>Bahnschrift SemiLight</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="type_LineEdit">
      <property name="font">
       <font>
        <family>Bahnschrift SemiLight</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="title_LineEdit">
      <property name="font">
       <font>
        <family>Bahnschrift SemiLight</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="source_LineEdit">
      <property name="font">
       <font>
        <family>Bahnschrift SemiLight</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="pages_LineEdit">
      <property name="font">
       <font>
        <family>Bahnschrift SemiLight</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="status_LineEdit">
      <property name="font">
       <font>
        <family>Bahnschrift SemiLight</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="contacts_LineEdit">
      <property name="font">
       <font>
        <family>Bahnschrift SemiLight</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>480</y>
     <width>281</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QPushButton" name="saveButton">
      <property name="font">
       <font>
        <family>Bahnschrift SemiLight</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(176, 196, 222);</string>
      </property>
      <property name="text">
       <string>Сохранить</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>38</x>
     <y>30</y>
     <width>291</width>
     <height>431</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Bahnschrift SemiLight</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ФИО автора</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Bahnschrift SemiLight</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Тип произведения</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <family>Bahnschrift SemiLight</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Название произведения</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>Bahnschrift SemiLight</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Источник (издание)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <family>Bahnschrift SemiLight</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Кол-во страниц</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="font">
       <font>
        <family>Bahnschrift SemiLight</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Статус</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="font">
       <font>
        <family>Bahnschrift SemiLight</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Контактные данные (тел.)</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>360</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiLight</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>(true или false)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>410</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiLight</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>(8XXXXXXXXXX)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
